docs/development.md

```diff
@@ -26,7 +26,22 @@ sholdn't be any):
 
 Install the runtime requirements using automation packages files:
```

Comment:
We can remove the "using automation packages files" now.

Reply:
done